src/app/core/feedback/feedback.service.js

```diff
@@ -56,16 +70,51 @@ const search = async (reqUser, queryParams, query) => {
 	const offset = page * limit;
 
 	// Query for feedback
-	const feedback = await Feedback.textSearch(query, null, limit, offset, sortArr, true, {
+	const feedback = await Feedback.textSearch(
+		query,
+		null,
+		limit,
+		offset,
+		sortArr,
+		true,
+		{
+			path: 'creator',
+			select: ['username', 'organization', 'name', 'email']
+		}
+	);
+
+	return util.getPagingResults(limit, page, feedback.count, feedback.results);
+};
+
+const getFeedbackById = async (feedbackId) => {
+	return Feedback.findOne({
+		_id: mongoose.Types.ObjectId(feedbackId)
+	}).populate({
 		path: 'creator',
-		select: ['name', 'email']
+		select: ['username', 'organization', 'name', 'email']
 	});
+};
 
-	return util.getPagingResults(limit, page, feedback.count, feedback.results);
+const updateFeedbackAssignee = async (feedbackId, assignee) => {
+	const feedback = await getFeedbackById(feedbackId);
+	feedback.assignee = assignee;
+	feedback.updated = Date.now();
+	await feedback.save();
```

Comment:
@sshah-asymmetrik since we're returning this `feedback` object to the controllers or other services ( as a public method ), let's favor a consistent return type of toJSON or toObject. Otherwise, we're might get unexpected values when the controller is attempting to render the service response as json.

Reply:
@brianghig @sshah-asymmetrik returning the mongoose object should be safe.  Since there is a `toJSON` function, it will be invoked automatically by express.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#tojson_behavior